{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--e0822689-7ec3-4809-a087-9aa5ad173ba7",
    "created": "2020-05-26T20:29:54.769996Z",
    "modified": "2020-05-26T20:29:54.769996Z",
    "name": "CVE-2018-6892",
    "description": "An issue was discovered in CloudMe before 1.11.0. An unauthenticated remote attacker that can connect to the \"\"CloudMe Sync\"\" client application listening on port 8888 can send a malicious payload causing a buffer overflow condition. This will result in an attacker controlling the program's execution flow and allowing arbitrary code execution."
}